applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -334,119 +330,28 @@ def PrepareModelPart(self):
             self._turbulence_model_solver.PrepareModelPart()
 
     def Initialize(self):
+        # If the solver requires an instance of the stabilized formulation class, set the process info variables
+        if hasattr(self, 'formulation'):
+            self.formulation.SetProcessInfo(self.GetComputingModelPart())
```

Comment:
nice, a functions called `SetProcessInfo` accepting a `ModelPart` as input :D

Reply:
Well, probably the name of the method is not so much self-explanatory.

What this method does is to set the values of the `StabilizedFormulation` auxiliary base class that need to be in the model part process info.

Maybe makes sense to open a future PR to rename it to something like `SetProcessInfoData` or `SetProcessInfoValues`.

Reply:
(added to the Fluid Dynamics Application project)